### PR TITLE
Oci ui include artifact list and artifact summary

### PR DESCRIPTION
**Pull request by jwangyangls**
Signed-off-by: Yogi_Wang <yawang@vmware.com>

### CODE REVIEW

File: src/portal/package-lock.json [hunk 1]
```diff
@@ -13681,7 +13681,7 @@
         },
         "serialize-error": {
             "version": "2.1.0",
-            "resolved": "http://registry.npmjs.org/serialize-error/-/serialize-error-2.1.0.tgz",
+            "resolved": "https://registry.npmjs.org/serialize-error/-/serialize-error-2.1.0.tgz",
             "integrity": "sha1-ULZ51WNc34Rme9yOWa9OW4HV9go="
         },
         "serialize-javascript": {
```
Comment (AllForNothing) [inline src/portal/package-lock.json]: any changes for package.json?

Comment (zhoumeina) [reply src/portal/package-lock.json]: If no change in package.json, why only this lock file changed?

File: src/portal/src/app/repository/artifact-list-page/artifact-list-page.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div>
+    <div class="breadcrumb" *ngIf="!withAdmiral">
+        <a (click)="goProBack()">{{'SIDE_NAV.PROJECTS'| translate}}</a>
+        <span class="back-icon"><</span>
+        <a (click)="watchGoBackEvt(projectId)">{{'REPOSITORY.REPOSITORIES'| translate}}</a>
+        <span *ngIf="referArtifactNameArray.length===1">&lt;<a (click)="backInitRepo()">{{repoName}}</a></span>
```
Comment (zhoumeina) [inline src/portal/src/app/repository/artifact-list-page/artifact-list-page.component.html]: Can this referArtifactNameArray be null or undefined?

Comment (jwangyangls) [reply src/portal/src/app/repository/artifact-list-page/artifact-list-page.component.html]: it has default  value  []

File: src/portal/package-lock.json (hunk 1 above)
Comment (jwangyangls) [reply src/portal/package-lock.json]: I have restore

**Pull request merged**